chainercv/datasets/camvid/camvid_dataset.py

```diff
@@ -0,0 +1,80 @@
+import glob
+import os.path as osp
```

Comment:
`import numpy as np` is allowed, but `import os.path as osp` is not allowed. Please use `os.path` directly.